docs/source/error_code_list.rst

```diff
@@ -415,11 +415,11 @@ Example:
     # Error: Dict entry 0 has incompatible type "str": "str"; expected "str": "int"  [dict-item]
     d: Dict[str, int] = {'key': 'value'}
 
-Check TypedDict items [typeddict-item]
---------------------------------------
+Check ``TypedDict`` items [typeddict-item]
```

Comment:
Please advise whether verbatim texts in section headers are useful. This renders as:

![image](https://user-images.githubusercontent.com/4455652/66528179-48c5e680-eaff-11e9-9e8a-80f439ad3396.png)

The side effect is also verbatim text in the table of contents:

![image](https://user-images.githubusercontent.com/4455652/66528209-5ed3a700-eaff-11e9-8759-06ec859458ae.png)

Reply:
I think it looks better without the heavy formatting. TBH I find most CSS styling for code blocks over the top, and red is particularly bad in a heading. I also don't care about the box (or shading in other CSS setups).

Reply:
Agree. IMO the best solution is simply use monospaced font for verbatim formatting like Python stdlib docs do that (and afaik Javadoc too). If you don't mind, I will also replace other occurences of verbatim text in section titles in this PR.